framework/src/modules/chain/block_processor_v2.js

```diff
@@ -95,22 +95,14 @@ const getBytes = block => {
 	);
 
 	const totalAmountBuffer = intToBuffer(
-		block.totalAmount.toString(),
+		block.totalAmount,
 		SIZE_INT64,
 		LITTLE_ENDIAN,
 	);
 
-	const totalFeeBuffer = intToBuffer(
-		block.totalFee.toString(),
-		SIZE_INT64,
-		LITTLE_ENDIAN,
-	);
+	const totalFeeBuffer = intToBuffer(block.totalFee, SIZE_INT64, LITTLE_ENDIAN);
```

Comment:
I think this error was due to the deserialization timing.
All the processor logic assumes block to be deserialized, so i think this is also should be the case